src/ape_accounts/_cli.py

```diff
@@ -56,41 +57,44 @@ def generate(alias):
         confirmation_prompt=True,
     )
     path.write_text(json.dumps(EthAccount.encrypt(account.key, passphrase)))
-
-    notify("SUCCESS", f"A new account '{account.address}' has been added with the id '{alias}'")
+    helper.log_success(f"A new account '{account.address}' has been added with the id '{alias}'")
 
 
 # Different name because `import` is a keyword
 @cli.command(name="import", short_help="Add a new keyfile account by entering a private key")
 @non_existing_alias_argument
-def _import(alias):
+@plugin_helper()
+def _import(helper, alias):
     path = container.data_folder.joinpath(f"{alias}.json")
     key = click.prompt("Enter Private Key", hide_input=True)
     try:
         account = EthAccount.from_key(to_bytes(hexstr=key))
     except Exception as error:
-        raise Abort(f"Key can't be imported: {error}")
+        helper.abort(f"Key can't be imported: {error}")
+        return
```

Comment:
Isn't this unnecessary?

Reply:
The `return` is yes. I just put it there to make my IDE happy

Reply:
Haha, okay

Reply:
I will remove the return and suppress the warning